### PR TITLE
GitHub greg branch

### PR DESCRIPTION
**Pull request by kosinovsky**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,6 +18,16 @@ OPTION(SPATH_LINK_STATIC "Default to static linking? (Needed for Cray)" OFF)
 
 # Find Packages & Files
 
+LIST(APPEND libspath_srcs
```
Comment (adammoody) [inline CMakeLists.txt]: I don't think you need any of the libspath_src lines in the top level CMakeLists.txt.  See if it still works if you delete these lines.

File: src/spath.c [hunk 2]
```diff
@@ -14,18 +15,18 @@
...
-  if (size > 0) {
-    ptr = malloc(size);
-    if (ptr == NULL) {
-      fprintf(stderr, "Failed to allocate %llu bytes @ %s:%d", (unsigned long long) size, file, line);
-    }
-  }
-  return ptr;
-}
-#define SPATH_MALLOC(X) spath_malloc(X, __FILE__, __LINE__);
+//void* spath_malloc(size_t size, const char* file, int line)
```
Comment (adammoody) [inline src/spath.c]: And go ahead and delete these lines, too for cleanup

Comment (CamStan) [reply src/spath.c]: Looks like there were a few more lines that were commented out. Those need to be cleaned up as well?

**Comment by kosinovsky** [comment]
Hi Cameron.
Yes, I already got these. They were in Adam’s second comment
Thank you,
Greg

File: src/spath.c [hunk 3]
```diff
@@ -1423,25 +1409,25 @@ Pretty print for TotalView debug window
 /* This enables a nicer display when diving on a path variable
  * under the TotalView debugger.  It requires TV 8.8 or later. */
 
-#include "tv_data_display.h"
+//#include "tv_data_display.h"
 
-static int TV_ttf_display_type(const spath* path)
-{
-  if (path == NULL) {
+//static int TV_ttf_display_type(const spath* path)
```
Comment (CamStan) [inline src/spath.c]: @kosinovsky: oops, yeah, sorry. I was referring to these lines that were commented out as well (between lines 1402-1433)

**Comment by kosinovsky** [comment]
Got it, thank you

**Pull request merged**